docker/entry.sh

```diff
@@ -29,6 +29,27 @@ map_dist() {
 	return 0
 }
 
+host_dist_matches_image_dist() {
+	if [ -z "$LB_DIST" ]; then
+		[ -f "$HOSTRELEASE" ] || die "You have to bind-mount /etc/os-release to the container's $HOSTRELEASE"
+		host_dist="$(lbdisttool.py -l --os-release $HOSTRELEASE | cut -d'.' -f1 )"
+	else
+		host_dist="$LB_DIST"
+	fi
+	image_dist="$(lbdisttool.py -l | cut -d'.' -f1 )"
+
+	if [[ -z $host_dist ]] ; then
```

Comment:
move that part immediately after you generated `host_dist`, before you even generate `image_dist`.

Reply:
done